Automattic/sensei
# Add tests for the courses listing availability and course creation

### Changes proposed in this Pull Request

* Add end-to-end tests for course creation.

### Testing instructions

* Run E2E tests: `$ npm run test:e2e`
* Check that "Create Courses › it should create a course" have been passed successfully.

### Screenshot / Video
